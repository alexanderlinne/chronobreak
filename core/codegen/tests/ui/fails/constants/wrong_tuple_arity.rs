#[macro_use]
extern crate chronobreak_derive;

constants![(0, 0)];

fn main() {}
