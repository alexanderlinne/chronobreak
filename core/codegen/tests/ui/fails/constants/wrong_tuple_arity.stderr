error: constants! expected a 3-tuple here:
 --> $DIR/wrong_tuple_arity.rs:4:13
  |
4 | constants![(0, 0)];
  |             ^^^^
